README.md

```diff
@@ -1,18 +1,44 @@
-# Getting Started with Create React App
+# CUBluePrint's New Website 
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Our new and updated website for CUBluePrint!
 
-## Available Scripts
+You can find our live website available at: [link](enter link here thanks)
+
+
+
+## Motivation for Project
+
+In 2021, we decided to build our club a brand new website with an improved UI and a more modern tech stack. The reason for this change was that our old website was built with __[insert outdated technology]__ and was becoming increasingly difficult for our exec team to maintain.
+
+To meet these challenges, our team designed a [new UI using Figma](https://www.figma.com/file/Qt7bGynNItu0hUvPu0EPGa/Draft-Website-Design?node-id=0%3A1) and is currently working on developing it using React, Typescript and styled components. 
+
+
+
+## Site Map
+
+Our site contains information about CUBlueprint's mission statement, our current projects, our team, how to get involved and sign up forms for volunteers and non-profit organizations. 
+
+The following is a visual map of how to navigate our website:
+
+### `INCLUDE SAMEE'S SITE MAP`
 
-In the project directory, you can run:
 
-### `yarn start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Tools
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+* TypeScript
+* React
+* Yarn
+* Styled Components
+
+
+## How to run the website in Development
+
+To run the project in development mode on a local server, run the following command inside the project directory:
+
+### `yarn start`
```

Comment:
I have to admit, I'm surprised by how good this looks. 👍

Reply:
ratio'd on my own PR <3

Reply:
LOL